diff --git a/node_modules/binance/lib/rest.js b/node_modules/binance/lib/rest.js
index fad2b40..8781895 100644
--- a/node_modules/binance/lib/rest.js
+++ b/node_modules/binance/lib/rest.js
@@ -367,7 +367,14 @@ class BinanceRest {
         this._setTimestamp(query);
         return this._makeRequest(query, callback, 'api/v3/account', 'SIGNED');
     }
-
+    accountSnapshot(query = {}, callback) {
+        if (_.isFunction(query)) {
+            callback = query;
+            query = {};
+        }
+        this._setTimestamp(query);
+        return this._makeRequest(query, callback, 'sapi/v1/accountSnapshot', 'SIGNED');
+    }
     myTrades(query = {}, callback) {
         if (_.isString(query)) {
             query = { symbol: query };
